tests/Service/ProgrammesAggregationService/FindDescendantClipsTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Service\ProgrammesAggregationService;
+
+use BBC\ProgrammesPagesService\Domain\Entity\Programme;
+
+/**
+ * @covers BBC\ProgrammesPagesService\Service\ProgrammesAggregationService::findDescendantClips
+ */
+class FindDescendantClipsTest extends AbstractProgrammesAggregationTest
+{
+    public function testFindDescendantClips()
+    {
+        $mockProgramme = $this->createMock(Programme::class);
+        $mockProgramme->method('getDbAncestryIds')->willReturn([11, 12]);
+
+        $this->mockRepository->expects($this->once())
+             ->method('findDescendantsByType')
+             ->willReturn([['type' => 'clip'], ['type' => 'clip']]);
```

Comment:
You're missing an assert on the arguments that get passed to `findDescendantsByType`.

Also please use the `$this->programmesFromDbData($dbData)` pattern for generating and testing the results like in every other Service test.